docs/design/dpu_support.md

```diff
@@ -0,0 +1,15 @@
+## DPU support
+
+With the emergence of [Data Processing Units](https://blogs.nvidia.com/blog/2020/05/20/whats-a-dpu-data-processing-unit/) (DPU) 
```

Comment:
First sentence needs to be rewritten, the 'which' doesn't work. Maybe something like:
With the emergence of [Data Processing Units](https://blogs.nvidia.com/blog/2020/05/20/whats-a-dpu-data-processing-unit/) (DPUs), NIC Vendors can now offer greater hardware acceleration capability, flexibility and security.

Reply:
Changed.